File: CTDK_Python/090/_090.py
x = float(input("Nhap x: "))
n = int(input("Nhap n: "))

s = 0
t = 1
m = 1
dau = -1
i = 1
while(i <= n):
    t = t * x
    m = m * i
    s = s + dau * t / m
    i = i + 1
    dau = -dau

print("Tong la: ", s)

File: CTDK/090/090.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main()
{
	float x;
	cout << "Nhap x: ";
	cin >> x;

	int n;
	cout << "Nhap n: ";
	cin >> n;

	float s = 0;
	int m = 1;
	float t = 1;
	int dau = -1;
	int i = 1;
	while (i <= n)
	{
		m = m * i;
		t = t * x;
		s = s + dau * t / m;
		dau = -dau;
		i++;
	}
	cout << "Tong la: " << s << endl;
	return 0;
}